site/content/book/v2/faq.md

```diff
@@ -0,0 +1,21 @@
+# Intro
+This file will work as a lightweight frequently asked questions before it grows into something unmanagable and needs to have better structure on it. For now this will offer a simple content adressing list an link to each question.
+We have sections of questions and questions underneath each section. They link directly to each question.
+
+## Cryptography
+1. [How kosher is it to do non encrypted messages?](#how-kosher-is-it-to-do-non-encrypted-messages) 
+
+******
+
+### How kosher is it to do non encrypted messages?
```

Comment:
This question raises two more questions for me:
1. Why are out of band didcomm invitations not signed?
2. Do out of band invitations fall under 1. of the two reasons listed? Are out of band invitations always considered public, even if you only share it with a single party?

Reply:
@TimoGlastra great questions. @dhh1128 maybe we can raise these in the next unsync meeting?

@TimoGlastra are these questions related specifically to this PR FAQ or more open ended?
Meaning they can become questions of their own?

My attempt and understanding
1. It is not nessecarily needed, as it is difficult to build a connection to another did that the one that is represented, and that endpoint is coming from the did itself, meaning that the trust that the did have controll over the endpoint is intrinsicly baked in? Or are there any other case im not seeing?

2. I would say yes to this, but I would love to hear others opinion